packages/frontend/src/features/sqlRunner/store/sqlRunnerSlice.ts

```diff
@@ -40,7 +44,8 @@ const initialState: SqlRunnerState = {
     projectUuid: '',
     activeTable: undefined,
     savedChartUuid: undefined,
-    name: 'Untitled SQL Query',
+    name: '',
```

Comment:
do we want to use `DEFAULT_NAME` here?

Reply:
No, we want this to by undefined until someone saves. 

Maybe it should be called NAME_PLACEHOLDER instead of default

Reply:
All good, you can ignore then 👍